core/src/main/scala/org/apache/spark/scheduler/cluster/ExecutorData.scala

```diff
@@ -34,5 +34,6 @@ private[cluster] class ExecutorData(
    override val executorHost: String,
    var freeCores: Int,
    override val totalCores: Int,
-   override val logUrlMap: Map[String, String]
-) extends ExecutorInfo(executorHost, totalCores, logUrlMap)
+   override val logUrlMap: Map[String, String],
+   override val workerUrl: Map[String, String]
+) extends ExecutorInfo(executorHost, totalCores, logUrlMap, workerUrl)
```

Comment:
Let me also ping @JoshRosen about this change since it is adding a new field to `ExecutorInfo`. 

@JoshRosen Does this change look good to you?